### ISSUE TITLE
Anyone getting Google Voice slapped or banned lately?

### ISSUE THREAD

**Issue by jodyheath** [brismuth/google-voice-bulk-texter#65]
Hello Everyone, I run a lotto pool and use the plugin for messaging people I know, personally. So for the last ~ 10 months or so I've been using this beloved plugin to send all my bulk texts. 

On Tuesday, I noticed only a few of my messages went out. And today, only 18 of 45 were delivered. Afterwards, I tried to bypass the plugin and send individual messages 30 min's after that failure, and got the same message, "Message Failed to Send".  I tried from both my desktop computer and my phone.

It seems my account was slapped again, today, like it was probably slapped on Tuesday.  My guess is, in about 12 - 24 hours I'll be able to send messages again. But I get the feeling Google may be tightening down on Google Voice a bit.

With this plugin, I have had pretty near perfect success over the last 6 months until this week.  Here is how I have been using it:

- I send  a bulk text x 45 recipients every Monday and Thursday.

- I send another bulk text x 35 recipients, either later on Monday & Thursday, or the following day.

- I send another bulk text x 45 recipients every Tuesday and Friday.

- So in summary, I send about 160 messages across 6 separate bulk texts per week.

First and foremost, I'm not blaming the plugin. Many of us know we are swimming in a bit of a grey area. But any ideas how to prevent this from happening?

Best,
 
 
--Jody

**Comment by EdWood110** [comment]
Hi Jody,

Our Google Voice account actually got suspended a few weeks ago. Your experience sounds a lot like yours.
We did contact Google and sent an email that we think we got wrongfully suspended and if we did something wrong, they should be so kind and explain it to us so we could avoid the problem in the future.
After one day we simply received an email that we did not do anything wrong, we did not violate any of their policies and the Google Voice account was live again.
However, we never received an explanation why it happened in the first place.

Good Luck!

**Comment by jodyheath** [comment]
Thanks for the feedback Ed. Can I ask for the email address you send your inquiry to?

--Jody

**Comment by jodyheath** [comment]
Update: so my google voice account was only banned from sending outbound messages.  I still received all inbound text during the perceived ban.  The ban was seemingly lifted tonight, about 48 hours after it started, without me contacting Google.  Timing was good, because it was that day of the week for me to send a big bulk text. I sent my bulk texts; however, after 18 messages, it stopped, and the remaining 17 recipients did not get sent (the dreaded "Message Failed to Send" notice).  

What was different this time, however, was last time I could not send any messages from the plugin or the regular google voice console after the perceived Google slap. This time, though, I can still send simple messages.  

These bulk texts, I have been sending, historically have all had one link (https://) in them, just as they have had for the last 10 months. After removing the link from the message, tonight, I was able to send it successfully the message. Not great news, as it seems Google Voice is really tightening down on messages with links in them.  

**But not so fast!, I put the link back in, and changed up some of the wording, and that message was also successfully sent.**  

And as a final test, I resent the same exact message (link included), and it was **not sent**.

So as it I can see it, while links may put you on the radar, the things that really get you slapped are sending messages too frequently, and sending too many identical messages.

I may have to start sending no more than 15 messages at a time.  And then I may need to change up the wording a bit. 

My other option is to: paste the phone numbers into Evernote on my desktop; open evernote on my iPhone; copy all the numbers into the messaging app; and turn off "group messaging" so no one sees everyone eases phone number. But that's a lot of steps to:o(

Verdict: it's probably still easier to just use the plugin, by sending smaller batches of texts, and changing up the wording to avoid falsely triggering GV's spam filters.

**Comment by EdWood110** [comment]
Hello Jody,

You can appeal the ban/suspension via this page.
[](https://support.google.com/accounts/contact/suspended)

All the best!

**Comment by brismuth** [comment]
@EdWood110 I've added a link to this thread from the support page (https://github.com/brismuth/google-voice-bulk-texter/blob/master/support.md), thanks for the tip.

**Comment by mwpettee** [comment]
@brismuth I am to am experiencing the same issue that @protectivediet mentioned . I sent 50 texts yesterday around 915pm EST and 50 texts today around 515pm EST, then when I fired off another 50 about an hour later, I received the same "`Google Voice bulk texter v1.3.2: Text failed. You can find support resources by opening the Google Voice Bulk Texter popup and clicking "Get Help" at the bottom. Error: "confirmSent" failed. When you click "ok" the page will refresh.`" message.  I tried to reduce the number to 10 recipients but still received the error. I am just sending a message for my boyfriend to some friends and clients that he has moved from one salon to another. Nothing that is bad or that even has links. 

Any suggestions?
Many thanks!
Mel

**Comment by mwpettee** [comment]
@brismuth ...ok...thank you for the response!

**Comment by Justintammy123** [comment]
My Google voice got suspended in error and i want to have access back to my Google voice number
